chandigarh/chandigarh.json

```diff
@@ -0,0 +1,152 @@
+    "description": "",
+    "eligibility": "",
+    "requiredDocuments": ""
+  },
+  {
+    "id": "1d810e6a-5359-4b79-90db-c6a0886a38f4",
+    "title": "Pension To The Disabled Persons",
+    "scheme_url": "http://chdsw.gov.in/index.php/scheme/pension-to-the-disabled-persons",
+    "applyLink": "http://chdsw.gov.in/uploads/media/1604736707-Pension_Disabled.pdf",
+    "description": "- Disability pension to disabled person is granted to Physically Handicapped person. Persons who are disabled to the extent of 40% and are residents of Union Territory of Chandigarh and fulfill the income criteria i.e. family income does not exceed Rs.1.50 lacs p.a. are eligible for grant of pension. The quantum of assistance is as under",
+    "eligibility": "",
+    "requiredDocuments": ""
+  },
+  {
+    "id": "c27eb8f9-a161-46d3-a1ac-17d8457404ae",
+    "title": "Pension To Widows And Destitute Women",
+    "scheme_url": "http://chdsw.gov.in/index.php/scheme/pension-to-widows-and-destitute-women",
+    "applyLink": "http://chdsw.gov.in/uploads/media/1604736234-Pension_WidowDestitute.pdf",
+    "description": "To provide financial assistance to Widows and Destitute Women.",
+    "eligibility": "The women of age group 18-60.\nResident of UT, Chandigarh for more than 3 years.\nMonthly family income of the applicant should not exceed Rs. 1.50 lacs annually.\n",
+    "requiredDocuments": "Residence proof of three years (Voter Card, Ration Card, Electricity Bill etc).\nDeath certificate of husband (In case of widow).\nAadhar Card.\nMobile Number.\n"
+  },
+  {
+    "id": "4f62b4a3-f63d-4afa-8636-f51775f043f3",
+    "title": "Old Age Pension",
+    "scheme_url": "http://chdsw.gov.in/index.php/scheme/old-age-pension",
+    "applyLink": "http://chdsw.gov.in/uploads/media/1602674701-Pension_Old.pdf",
+    "description": "To provide financial assistance to elderly people.",
+    "eligibility": "Monthly family income of the applicant should not exceed Rs. 1.50 lacs annually.\nResident of UT, Chandigarh for more than 3 years.\nAge 60 years and above.\n",
+    "requiredDocuments": "Two passport size photographs.\nResidence proof of three years (Voter Card, Ration Card, Electricity Bill etc).\nAadhar Card.\nMobile Number.\n"
+  },
+  {
+    "id": "6e38687d-0799-45fe-aab5-781d8340e5f1",
+    "title": "ICDS (Integrated Child Development Scheme)",
+    "scheme_url": "http://chdsw.gov.in/index.php/scheme/ICDS",
+    "applyLink": "http://chdsw.gov.in/uploads/media/1626332162-Activity_Book_for_3-4_years_Children_1.pdf",
+    "description": "Integrated Child Development Services (ICDS)",
+    "eligibility": "",
+    "requiredDocuments": ""
+  },
+  {
+    "id": "523bfbd1-a76b-4899-94f8-f0d24fa082ef",
+    "title": "Sponsorship Scheme -A financial assistance to promote family based care with a view to improve the quality of life of children",
+    "scheme_url": "http://chdsw.gov.in/index.php/scheme/Sponsorship-Scheme",
+    "applyLink": "",
```

Comment:
@Karthikreddy0165 There are no applyLinks, can we check if we missed this in scrapping?